templates/base.html

```diff
@@ -145,8 +143,6 @@
             <ul class="footer-items">
                 <li><a href="{% url 'privacy' %}">Privacy</a></li>
                 <li><a href="{% url 'terms' %}">Terms of Use</a></li>
-                <li><a href="{% url 'resources' %}">Resources</a></li>
```

Comment:
Should we delete the resources and FAQ pages, or can they be accessed from another page?